x/interchainstaking/keeper/msg_server.go

```diff
@@ -237,12 +237,12 @@ func (k msgServer) GovCloseChannel(goCtx context.Context, msg *types.MsgGovClose
 			)
 	}
 
-	_, cap, err := k.Keeper.IBCKeeper.ChannelKeeper.LookupModuleByChannel(ctx, msg.PortId, msg.ChannelId)
```

Comment:
is `cap` a keyword? I dislike `c` as a variable name, it's somewhat ambiguous. Can we make it `capability` instead?

Reply:
`cap` a keyword (for "capacity").

I'll modify to "capability"